bin/runtime/src/lib.rs

```diff
@@ -912,6 +919,7 @@ construct_runtime!(
         Identity: pallet_identity = 20,
         CommitteeManagement: pallet_committee_management = 21,
         Proxy: pallet_proxy = 22,
+        Operations: pallet_operations = 23,
```

Comment:
can we put some large number here? We will most likely remove this pallet in the future, right?

Reply:
Whether we remove or not I don't know, but I think putting a large index makes sense.